be/src/vec/data_types/data_type_factory.cpp

```diff
@@ -32,7 +32,7 @@ DataTypePtr DataTypeFactory::create_data_type(const doris::Field& col_desc) {
     }
 
     if (col_desc.is_nullable() && nested) {
-        return std::make_shared<DataTypeNullable>(std::move(nested));
+        return std::make_shared<DataTypeNullable>(nested);
```

Comment:
why do not use `std::move` ?

Reply:
Because there is no constructor `DataTypeNullable(const DataTypePtr&&)` in class `DataTypeNullable`.